### PR TITLE
Shorten generated ConfigVersion names

### PR DESCRIPTION
**Pull request by dprince**
OSPK8-473 #resolve done

### CODE REVIEW

File: api/v1beta1/openstackdeploy_types.go [hunk 1]
```diff
@@ -48,6 +48,8 @@ const (
 	DeployCondReasonCVUpdated ConditionReason = "ConfigVersionUpdated"
 	// DeployCondReasonConfigVersionNotFound - error finding ConfigVersion
 	DeployCondReasonConfigVersionNotFound ConditionReason = "ConfigVersionNotFound"
+	// DeployCondReasonConfigGeneratorNotFound - error finding ConfigVersion
```
Comment (stuggi) [inline api/v1beta1/openstackdeploy_types.go:51]: ConfigVersion => ConfigGenerator

File: pkg/openstackconfigversion/git_util.go [hunk 2]
```diff
@@ -204,14 +204,14 @@ func SyncGit(
 					//fmt.Println(buffer.String())
```
Comment (stuggi) [inline pkg/openstackconfigversion/git_util.go:204]: could remove this ?

**Comment by dprince** [comment]
Seem to be hitting an issue with this now...

**Comment by dprince** [comment]
The associated dev-tools patch for this is here: https://github.com/openstack-k8s-operators/osp-director-dev-tools/pull/332

**Pull request closed without merging**